client/lib/domains/resolve-domain-status.js

```diff
@@ -206,41 +223,117 @@ export function resolveDomainStatus(
 					statusText: translate( 'Action required' ),
 					statusClass: 'status-error',
 					status: translate( 'Verify email' ),
+					noticeText: translate(
+						'We sent you an email at %(email)s to verify your contact information. Please complete the verification or your domain will stop working.',
```

Comment:
It looks that "resend email" and "change email address" links are missing here, but maybe we have discuss to remove it. I don't remember, sorry 😅

Reply:
Not really, I forgot to add it, sorry 😄 
Actually, I changed the messaging a bit: 

![image](https://user-images.githubusercontent.com/18705930/148466462-87cab9b8-386d-4d7e-96bd-87f7b9bf9303.png)

I removed the domain owner email from it. To retrieve it, we would have to perform an additional whois request. 
That wouldn't be a good solution for the "All my domains" page, nor for a site with many domains.
I also removed the "resend email" link, since I don't think that's sent by us. 😬